### PR TITLE
Add rubygems_version to version model

### PR DESCRIPTION
**Pull request by sonalkr132**
[Issue comment](https://github.com/rubygems/rubygems.org/issues/1037#issuecomment-219414615)

I have not added rubygems_version to view because there does not seem to be agreement about it. Should I add that as well?

### CODE REVIEW

File: lib/tasks/gemcutter.rake [hunk 1]
```diff
@@ -79,4 +79,24 @@ namespace :gemcutter do
       puts "Done."
     end
   end
+
+  namespace :rubygems_version do
+    desc "Backfill gem versions with rubygems_version."
+    task backfill: :environment do
+      without_rubygems_version = Version.where(rubygems_version: nil)
+      mod = ENV['shard']
```
Comment (sonalkr132) [inline lib/tasks/gemcutter.rake]: I added this just cause rake task to [backfill metadata](https://github.com/rubygems/rubygems.org/pull/1276/files#diff-057c0b82f700094d44295e5d85fc7a97R67) had it. Let me know if it is unnecessary.

File: app/models/version.rb [hunk 2]
```diff
@@ -204,6 +204,7 @@ def update_attributes_from_gem_specification!(spec)
       metadata: spec.metadata || {},
       requirements: spec.requirements,
       built_at: spec.date,
+      rubygems_version: (spec.required_rubygems_version || '').to_s,
```
Comment (dwradcliffe) [inline app/models/version.rb]: Any reason to not use `spec.required_rubygems_version.to_s`?

File: lib/tasks/gemcutter.rake (hunk 1 above)
Comment (dwradcliffe) [inline lib/tasks/gemcutter.rake]: This is good to leave. It allows us to run the backfill in multiple parallel tasks.

**Comment by dwradcliffe** [comment]
Let's not add anything to the view yet. We can get all the data backfilled and make sure it's working for the APIs first.

One small comment and this needs a rebase. Otherwise looking good!

File: app/models/version.rb (hunk 2 above)
Comment (sonalkr132) [inline app/models/version.rb]: I used `|| ''` cause thats what existed [on bundler-api repo](https://github.com/bundler/bundler-api/blob/7bf5af30c5079e4a99ab0e2646c023d41380f7ad/lib/bundler_api/update/gem_db_helper.rb#L87). I suppose it is for gemspec which have not specified `required_rubygems_version`? Checkout this commit: https://github.com/bundler/bundler-api/commit/ee311886c95da0d01cb4f8a6c2a85b1440c39402

Comment (dwradcliffe) [inline app/models/version.rb]: Ah I see. Well `nil.to_s == ""` so let's change this to match the line below. Better to keep similar style in this repo/method.

Comment (sonalkr132) [inline app/models/version.rb]: Makes sense. Updated :+1:

**Comment by dwradcliffe** [comment]
👍

**Comment by dwradcliffe** [comment]
The backfill has completed and we have data for 100% of versions now. ✨

**Pull request merged**